### PR TITLE
Feat/gtw 244/pending approval org page

### PR DESCRIPTION
**Pull request by andrefelizardo**
[Feature]

image

Waiting for approval page dialog and button.

**Tech Debt**
- I tried for days to implement a way to get the translation string text and append a React component to make the _email_ and _Discord_ links. To respect the deadline (in the sense that we aren't using translation on production) I added the text directly into the components.

### CODE REVIEW

File: src/components/features/daos/view/dao-header.tsx [hunk 1]
```diff
@@ -6,12 +6,17 @@ import { AvatarFile } from '@/components/atoms/avatar-file';
 import { FollowButtonDAO } from '@/components/atoms/buttons/follow-button-dao';
 import { ShareButton } from '@/components/atoms/buttons/share-button';
 import { ReadMore } from '@/components/atoms/read-more-less';
+// import TextWithParagraphs from '@/components/atoms/text-with-paragraphs/text-with-paragraphs';
```
Comment (rafagomes) [inline src/components/features/daos/view/dao-header.tsx:9]: remove commented code

**Review by boscocg — APPROVED** [review]

File: src/components/features/daos/view/dao-header.tsx [hunk 2]
```diff
@@ -96,18 +223,29 @@ export function DaoHeader({ followCount, onFollow, onUnfollow }: Props) {
...
+                label="Waiting for approval"
+                size="small"
+              />
+            )}
+            {dao.categories && (
+              <Stack direction="row" gap={1} sx={{ mt: 12 / 8 }}>
+                {dao.categories.map((category) => {
+                  const label = categoriesMap.get(category) ?? category;
+                  const formattedLabel =
+                    label.charAt(0).toUpperCase() + label.slice(1);
```
Comment (rafagomes) [inline src/components/features/daos/view/dao-header.tsx:241]: stead of doing this in JS, why not use text-transform in CSS? capitalize.
https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

**Review by rafagomes — CHANGES_REQUESTED** [review]

File: src/components/features/daos/view/dao-header.tsx (hunk 2 above)
Comment (andrefelizardo) [reply src/components/features/daos/view/dao-header.tsx:241]: Legacy

File: src/components/features/daos/view/dao-header.tsx (hunk 1 above)
Comment (andrefelizardo) [reply src/components/features/daos/view/dao-header.tsx]: Removed

**Review by rafagomes — APPROVED** [review]

**Pull request merged**